crowdaction/main.go

```diff
@@ -19,6 +19,9 @@ import (
 
 const (
 
+	//do not send back the code/password, just an indication it's needed
+	codeRequired = "yes"
```

Comment:
Maybe `"required"` would be the better value, since the JSON field is called `code`, so the context is a bit unclear.
(`"code":"yes"` vs `"code":"required"`)